docs/our_work/ds278_Assurance_Research_Path.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 'DS_278: Assurance Research Path'
```

Comment:
I wouldnt give the name of it the DS code, cause this is public facing and that doeasnt mean anything to anyone oputside the team + just clutters up the titles